app/views/settings/_org_two_factor_auth.html.erb

```diff
@@ -0,0 +1,26 @@
+<%- # License: AGPL-3.0-or-later WITH Web-Template-Output-Additional-Permission-3.0-or-later -%>
+<header class='pane-header'>
+	<h3>Organization Two Factor Authentication</h3>
+</header>
+
+<div class='pane-inner'>
+    <form autosubmit method='put' action='<%= nonprofit_path(@nonprofit) %>' data-reload>
+      <p class='pastelBox--yellow u-padding--10 u-marginBottom--20'>Two factor authentication adds an extra layer of security to users' accounts by requiring a one-time password when they log in. To enable, check the option below. Unchecking will not disable it for users that have it enabled, it will just stop requring it to be enabled.</p>
+      <div class='field'>
+        <input type='hidden' name='nonprofit[require_two_factor]' value='false'>
+        <input
+            type='checkbox'
+            name='nonprofit[require_two_factor]'
+            value='true'
+            <%= 'checked=checked' if @nonprofit.require_two_factor? %>
+            <%= 'disabled' unless current_role?([:nonprofit_admin,:super_admin]) %>
+            id='users-two-factor-input'>
+        <label class='two-factor-label' for='users-two-factor-input'>Require two factor authentication for all users.</label>
+      </div>
```

Comment:
@djohnston32 I'm curious why do we have both this hidden input and the checkbox input?